crmsh/cibconfig.py

```diff
@@ -3254,6 +3254,10 @@ def has_cib_changed(self):
             return False
         return self.modified_elems() or self.remove_queue
 
+    def ensure_cib_updated(self):
+        if not self.has_cib_changed():
```

Comment:
I suggest to add another check, like
`if options.interactive and not self.has_cib_changed():`
Since it has no such issue under non-interactive mode:)